jcg_testcases/src/main/resources/js/AnonymousFunctions.md

```diff
@@ -0,0 +1,65 @@
+# Anonymous Functions
+Anonymous functions, also called Lambda functions, are functions that are not bound to an identifier.
+
+## AF1
+[//]: # (MAIN: global)
+Test the use of an anonymous function as an argument to another function.
+
+```json
+{
+  "nodes": [
+    "<global>", "AF1.foo", "AF1.<anonymous:6>"
+  ],
+  "directLinks": [
+    ["<global>", "AF1.foo"],
+    ["AF1.foo", "AF1.<anonymous:6>"]
+  ],
+  "indirectLinks": []
+}
+```
+```js
+// af/AF1.js
+
+function foo(f) {
+    return f(1);
+}
+
+foo(function(x) {
```

Comment:
Might want to reformat the code here a little to make it easier to read and to not have multiple constructs on one line? I.e., start the argument in the next line instead of the one where the call is?